src/Containers/nested_iterator.jl

```diff
@@ -47,8 +55,18 @@ Create a [`NestedIterator`](@ref).
 
 ## Example
 
-```julia
-nested(1:2, ["A", "B"]; condition = (i, j) -> isodd(i) || j == "B")
```

Comment:
Here's one example where changing to a doctest caught some old syntax that hadn't been updated.